positional_goal_tolerance : 0.02
planner_type: "nonholonomic"
degrees_of_freedom: 3
m_arm: 5.0
m_base: 5.0
m_rot: 0.5
k_geo_col: 0.1
k_fin_col: 0.1
l_offset: "0.5"
goal_indices: 
  - 0
  - 1
  - 2
root_link: "world"
urdf_source: "/robot_description"
end_effector_link: 'ee_link'
orientation_helper_link: "base_link"
end_links: 
  - "ee_link"
num_sphere_obs : 0
num_box_obs : 0
num_plane_constraints: 0
angular_goal_tolerance: 0.1
collision_bodies:
  radius_body_ee_link: 0.2
collision_links: 
  - "ee_link"
velocity_scaling: 1.0
velocity_limits:
  upper:
        - 1.0
        - 1.0
  lower:
        - -1.0
        - -1.0
